src/test/ovmsconfig_test.cpp

```diff
@@ -329,6 +330,609 @@ TEST_F(OvmsConfigDeathTest, negativeMissingDashes) {
+}
+
+TEST_F(OvmsConfigDeathTest, hfBadEmbeddingsGraphParameter) {
+    char* n_argv[] = {
+        "ovms",
+        "--pull",
+        "--source_model",
+        "some/model",
+        "--model_repository_path",
+        "/some/path",
+        "--task",
+        "embeddings",
+        "--max_num_batched_tokens",
+        "145",
+    };
+    int arg_count = 10;
+    EXPECT_EXIT(ovms::Config::instance().parse(arg_count, n_argv), ::testing::ExitedWithCode(OVMS_EX_USAGE), "task: embeddings - error parsing options - unmatched arguments : --max_num_batched_tokens, 145,");
+}
+
+TEST_F(OvmsConfigDeathTest, hfBadTextGenGraphParameter) {
+    char* n_argv[] = {
+        "ovms",
+        "--pull",
+        "--source_model",
+        "some/model",
+        "--model_repository_path",
+        "/some/path",
+        "--task",
+        "text_generation",
+        "--normalize",
+        "true",
+    };
+    int arg_count = 10;
+    EXPECT_EXIT(ovms::Config::instance().parse(arg_count, n_argv), ::testing::ExitedWithCode(OVMS_EX_USAGE), "task: text_generation - error parsing options - unmatched arguments : --normalize, true,");
+}
+
+TEST_F(OvmsConfigDeathTest, hfBadTextGraphParameterName) {
+    char* n_argv[] = {
+        "ovms",
+        "--pull",
+        "--source_model",
+        "some/model",
+        "--model_repository_path",
+        "/some/path",
+        "--min_num_batched_tokens",
+        "145",
+    };
+    int arg_count = 8;
+    EXPECT_EXIT(ovms::Config::instance().parse(arg_count, n_argv), ::testing::ExitedWithCode(OVMS_EX_USAGE), "task: text_generation - error parsing options - unmatched arguments : --min_num_batched_tokens, 145,");
+}
+
+TEST_F(OvmsConfigDeathTest, hfBadRerankGraphParameterName) {
+    char* n_argv[] = {
+        "ovms",
+        "--pull",
+        "--source_model",
+        "some/model",
+        "--model_repository_path",
+        "/some/path",
+        "--task",
+        "rerank",
+        "--min_doc_length",
+        "1400",
+    };
+    int arg_count = 10;
+    EXPECT_EXIT(ovms::Config::instance().parse(arg_count, n_argv), ::testing::ExitedWithCode(OVMS_EX_USAGE), "task: rerank - error parsing options - unmatched arguments : --min_doc_length, 1400,");
+}
+
+TEST_F(OvmsConfigDeathTest, hfBadTextGenGraphParameterName) {
+    char* n_argv[] = {
+        "ovms",
+        "--pull",
+        "--source_model",
+        "some/model",
+        "--model_repository_path",
+        "/some/path",
+        "--task",
+        "text_generation",
+        "--min_num_batched_tokens",
+        "145",
+    };
+    int arg_count = 10;
+    EXPECT_EXIT(ovms::Config::instance().parse(arg_count, n_argv), ::testing::ExitedWithCode(OVMS_EX_USAGE), "task: text_generation - error parsing options - unmatched arguments : --min_num_batched_tokens, 145,");
+}
+
+TEST_F(OvmsConfigDeathTest, hfBadEmbeddingsGraphParameterName) {
```

Comment:
Duplicates test from L380?

Reply:
The *Name tests are with good parameters but the names of the parameters for specific graph is wrong. So no duplicate.